src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -1,11 +1,10 @@
-<nav class="navbar navbar-default navbar-main">
+<nav class="navbar navbar-default navbar-pf">
   <!-- Brand and toggle get grouped for better mobile display -->
 
   <div class="navbar-header tc_logo_component">
     <a class="navbar-brand"
        href="#">
-      <img src="assets/Ceph_Logo_Standard_RGB_White_120411_fa.png"
-           alt="Ceph">
+      <img src="/assets/branding/RHCSD-Masthead-Logo.svg" width=100%  alt="Red Hat Ceph Storage">
```

Comment:
If we use contants.ts `{{ PRODUCT_NAME }}`

Reply:
ack